layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,207 @@
+      <li>Jordan Edwards</li>
+      <li>Julius Johnson</li>
+      <li>Kajieme Powell</li>
+      <li>Keith Childress</li>
+      <li>Keith Lamont Scott</li>
+      <li>Kendrec McDade</li>
+      <li>Kwame Jones</li>
+      <li>Laquan McDonald</li>
+      <li>Larry Jackson Jr</li>
+      <li>Manuel Loggins Jr</li>
+      <li>Mario Woods</li>
+      <li>Maurice Granton</li>
+      <li>Michael Brown</li>
+      <li>Michael Dean</li>
+      <li>Mike Ramos</li>
+      <li>Miles Hall</li>
+      <li>Natosha McDade</li>
+      <li>Patrick Harmon</li>
+      <li>Philando Castile</li>
+      <li>Quintonio LeGrier</li>
+      <li>Ramarley Graham</li>
+      <li>Robert Lawrence White</li>
+      <li>Rumain Brisbon</li>
+      <li>Ryan Twyman</li>
+      <li>Samuel David Mallard</li>
+      <li>Samuel DuBose</li>
+      <li>Sandra Bland</li>
+      <li>Sean Reed</li>
+      <li>Stephon Clark</li>
+      <li>Steven Demarco Taylor</li>
+      <li>Tamir Rice</li>
+      <li>Terence Crutcher</li>
+      <li>Terrance Franklin</li>
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li><a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> names not listed here whose lives were taken by violence and brutality</li>
+    </ul>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+    <p>In Node.js, we're going to be working on supporting Black members of the community. If we can help - by using our collective platforms to amplify your words, by helping you in your career or in contributing to open-source, or in any other way - we are here for you.</p>
```

Comment:
**As an aside I admit I lack the context,** but I would be mindful of the fact that this whole paragraph reads like you think Black folks need your charity. It's subtle. I think that without recognising systemic privileges and that's the reason why you're all here and they're not is the imperative fact to not ignore.

Reply:
Particularly bc I'm looking at the names below....... **As an aside I admit I lack the context,** but I would be mindful of whatever you're discussing as mentorship unless you have to. I am very sensitive to the fact that a lot of this language implicitly says that Black people are in need of special help. Instead, it's that they haven't received the systematic privilege and the opportunities.

Reply:
Particularly bc I'm looking at the names below...... Might be good to be more explicitly recognise your own make up and even list your demographics here.

Reply:
```suggestion
    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change that, but we can change how we proceed moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and by financially supporting work from Black creators.
The labor to dismantle white supremacy does not fall on Black folks; it falls on the rest of us. If there are Black folks looking for collaboration, amplification, mentorship, guidance, or anything sort of support, we're here for you (see below).</p>
```

I reached out to @tatianamac on how to meaningfully write this and we workshopped this together, largely with her driving.

Reply:
I like it!. A few questions:

> We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members,

I interpreted this as if we (project) already are doing those things. Maybe make it more explicit that we are committing ourselves to doing that moving forward ("We are committing ourselves to..." or "Moving forward, we will be dedicated to...")? If we are doing those things already, please disregard this question.

> and by financially supporting work from Black creators

Is this referring to us individually financially supporting, or with the project supporting via the already available means like travel fund? Or did we get a budget approved with OpenJS Foundation/will be pushing to get a budget approved?

I'm fine either way, just think we need to clarify as it is open for interpretation.

Reply:
>Is this referring to us individually financially supporting, or with the project supporting via the already available means like travel fund? Or did we get a budget approved with OpenJS Foundation/will be pushing to get a budget approved?

Referring to us individually. FWIW the Travel Fund is no longer the project's but the CPC's. Foundation budget would need to go through the CPC - happy to do that work.

Reply:
@bnb `anything sort of support` → `any sort of support`?

Reply:
Resolving as we have updated the copy. please add additional comments below.

We are likely going to try shipping this later today, so it might be better to wait on any non-blocking changes until after this lands and fix in a follow up PR